lib/ret/avatar_listing.ex

```diff
@@ -60,6 +62,22 @@ defmodule Ret.AvatarListing do
     end
   end
 
+  defp avatar_listing_to_collapsed_files(%{parent_avatar_listing: nil} = a), do: a |> Map.take(Avatar.file_columns())
+
+  defp avatar_listing_to_collapsed_files(%{parent_avatar_listing: p} = a) do
```

Comment:
nit: use full names instead of `p` and `a`. Makes for easier reading and grepping.